dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1821,7 +1821,8 @@ FROM
     ('ryu-ryujin', 'ethereum', 'RYU ', 0xca530408c3e552b020a2300debc7bd18820fb42f, 18),
     ('biao-biaoqing', 'ethereum', 'BIAL ', 0x9fd9278f04f01c6a39a9d1c1cd79f7782c6ade08, 9),
     ('matt-matt-furie', 'ethereum', 'MATT ', 0x790814Cd782983FaB4d7B92CF155187a865d9F18, 9),
-    ('spurdo-spurdo-on-eth', 'ethereum', 'SPURDO ', 0x3927fb89f34bbee63351a6340558eebf51a19fb8, 18)
+    ('spurdo-spurdo-on-eth', 'ethereum', 'SPURDO ', 0x3927fb89f34bbee63351a6340558eebf51a19fb8, 18),
+    ('alph-alephium', 'ethereum', 'ALPH', 0x590f820444fa3638e022776752c5eef34e2f89a6, 18)
```

Comment:
Other tokens seems to have a space after the ticker name. We maybe should have one as well?

Reply:
hmm this actually seems like a missed bug, can we remove the space on those few rows instead? looks like ~10-15 lines have it